--- freelords/src/common/mapdef.cpp.orig	Wed Jan 16 17:24:34 2002
+++ reelords/src/common/mapdef.cpp	Tue Sep  3 20:53:14 2002
@@ -371,7 +371,7 @@
 			// and all variations of this tile
 			for (int k = 0; k < 6; k++)
 			{
-				sprintf(fname, "../pic/map/default/%s_%i_%i.png", name, j, k);
+				sprintf(fname, "/usr/X11R6/share/freelords/default/%s_%i_%i.png", name, j, k);
 
 				SDL_Surface* newpic = IMG_Load(fname);
 				if (newpic != 0)
--- freelords/src/graphic/hero_offer.cpp.orig	Mon Aug  5 11:55:16 2002
+++ freelords/src/graphic/hero_offer.cpp	Wed Sep 18 16:44:40 2002
@@ -61,8 +61,8 @@
 			(MSG_CALLBACK_OBJ)&Hero_offer::b_reject_clicked);
 
 	// generate a name depending on sex of hero
-	if (d_gender == MALE) system("./hero_names.pyw 1 m");
-	else system("./hero_female_names.pyw 1 f");
+	if (d_gender == MALE) system("/usr/X11R6/share/freelords/hero_names.pyw 1 m");
+	else system("/usr/X11R6/share/freelords/hero_female_names.pyw 1 f");
 	
     char new_name[80];
 	std::ifstream name_file("new_name");
