--- freelords-0.3.7/src/path.cpp~	2006-03-01 22:05:35.000000000 +0100
+++ freelords-0.3.7/src/path.cpp	2006-11-11 14:07:55.000000000 +0100
@@ -71,7 +71,7 @@
     }
 
     retval &= helper->openTag("path");
-    retval &= helper->saveData("size", size());
+    retval &= helper->saveData("size", (unsigned int)size());
     retval &= helper->saveData("x", sx.str());
     retval &= helper->saveData("y", sy.str());
     retval &= helper->closeTag();
